src/vite-env.d.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference types="vite/client" />
+/// <reference types="vite-plugin-comlink/client" />
```

Comment:
do we need these comment lines ?

Reply:
Yes, it is a part of vite configuration.